src/ol/control/ZoomSlider.js

```diff
@@ -26,10 +26,12 @@ const Direction = {
 
 /**
  * @typedef {Object} Options
+ * @property {boolean} [constrainResolution=true] Constrain zoom level to an integer.
  * @property {string} [className='ol-zoomslider'] CSS class name.
  * @property {number} [duration=200] Animation duration in milliseconds.
  * @property {function(import("../MapEvent.js").default)} [render] Function called when the control
  * should be re-rendered. This is called in a `requestAnimationFrame` callback.
+ * @property {HTMLElement|string} [target] Specify a target if you want the control to be
```

Comment:
It looks like there should be some more words in this description.

Reply:
You are correct.